packages/eslint-plugin-office-addins/src/rules/load-object-before-read.ts

```diff
@@ -92,8 +106,8 @@ export = {
           }
 
           if (
```

Comment:
Should include the '!propertyName' test in this same 'if' statement.  Can also move the next 'if' statement to right after setting the 'node' value to avoid the extra code processing when we can.